projects/packages/publicize/src/js/classic-editor-connections.js

```diff
@@ -125,7 +125,7 @@ jQuery( function ( $ ) {
 				unsupportedConnections++;
 
 				isInvalid = true;
-			} else if ( connection.status === 'broken' ) {
+			} else if ( [ 'broken', 'must_reauth' ].includes( connection.status ) ) {
 				brokenConnections++;
 
 				isInvalid = true;
```

Comment:
The connections that need reauth are treated as broken from the point of view of notifying the user, but they're not actually broken, and should still be able to be used like a valid connection. `must_reauth` used to be because the API had changed, but now it is if the connection will expire at some point soon.

Reply:
No, I updated the messaging on the similar lines that you did in #41494

Reply:
Yeah it's not the messaging, it's the fact that it sets `isInvalid` to true. Doesn't that mean we'll disable it?

Reply:
Ah yes, that is right. I have fixed it in eb2611b66d2b854253692993be9ec18a2110a949